client/components/Editor/index.jsx

```diff
@@ -214,35 +277,28 @@ class Editor extends Component {
                 onClick={this.onClickScenarioAction}
             />
         );
+        const editTabMenu = Object.keys(this.state.tabs).map(tabType => {
```

Comment:
Switching to using the `tabs` state object saves some copy pasting and also needing to check if it's a new scenario or not.